internal/core/domain/stats_model.go

```diff
@@ -0,0 +1,31 @@
+package domain
+
+// Withdrawal is used to follow funds withdrawal statistics
+type Withdrawal struct {
+	ID              uint `badgerhold:"key"`
+	AccountIndex    int
+	BaseAmount      uint64
+	QuoteAmount     uint64
+	MillisatPerByte int64
+	Address         string
+}
+
+// Deposit is used to follow deposit statistics made by operator
+type Deposit struct {
```

Comment:
this should define other 2 fields (asset and amount) according to the previous suggestion on proto definition.

Reply:
Done